scss/_settings_grid.scss

```diff
@@ -5,15 +5,29 @@
 $grid-max-width: 80rem !default;
 $grid-columns-small: 4 !default;
 $grid-columns-medium: 6 !default;
+$grid-columns-medium-new: 4 !default;
 $grid-columns: 12 !default;
+
 $threshold-4-6-col: $breakpoint-small !default;
+$threshold-4-small-4-med-col: $breakpoint-small !default;
 $threshold-6-12-col: $breakpoint-large !default;
+$threshold-4-med-12-col: $breakpoint-large !default;
 
 $grid-column-prefix: 'col-' !default;
+$grid-row-class: 'row' !default;
+$grid-row-4-cols-class: '#{$grid-row-class}--4-cols' !default;
```

Comment:
If we don't have a better semantic name for it, I'd suggest calling it `row--medium-4-cols` or `row--4-cols-medium` to at least make it clearer in the name that 4 cols change only affects medium screen size.

Reply:
Updated to `row--4-cols-medium`